Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageLayoutOptionsResolver.swift

```diff
@@ -73,15 +84,15 @@ open class _ChatMessageLayoutOptionsResolver<ExtraData: ExtraDataTypes> {
         if isLastInSequence && !message.isSentByCurrentUser && !channel.isDirectMessageChannel {
             options.insert(.authorName)
         }
-        if message.quotedMessage?.id != nil {
+        if isQuoted(message) {
             options.insert(.quotedMessage)
         }
         if message.isRootOfThread || message.isPartOfThread {
             options.insert(.threadInfo)
             // The bubbles with thread look like continuous bubbles
             options.insert(.continuousBubble)
         }
-        if !message.reactionScores.isEmpty && channel.config.reactionsEnabled {
+        if hasReactions(channel, message, appearance) {
```

Comment:
nit: would be great to have a unit-test that checks `.reactions` option is not included if for all reactions icons are missing (`ChatMessageLayoutOptionsResolver_Tests`)